tools/benchmark/binsize/README.md

```diff
@@ -0,0 +1,7 @@
+# icu_benchmark_binsize [![crates.io](http://meritbadge.herokuapp.com/icu_benchmark_binsize)](https://crates.io/crates/icu_benchmark_binsize)
+
+
```

Comment:
Suggestion: Please document this feature here, and in ` tools/benchmark/README.md

Reply:
Documented in tools/benchmark/README.md and linked to it in tools/benchmark/binsize/README.md.

Reply:
Hmm, a subsequent 'cargo make generate-readmes' threw out the change I made to benchmark/binsize/README.md.

Reply:
suggestion (follow-up, optional): Ah, I believe the `generate-readmes` tool reads the mod.rs to generate this file. I'm not sure about this since it's a `main.rs`. It would probably be worth figuring out, but shouldn't block landing. It's probably fine for now that `tools/benchmark/README.md` has the documentation, plus this was an existing issue.